### DISCUSSION TITLE
kPageTopicsBatchAnnotations removed in v115

### DISCUSSION THREAD

**Discussion by uazo** [uazo/bromite-buildtools#304] [General]
Hi @chirayudesai

I've seen the comments in https://review.calyxos.org/c/CalyxOS/platform_external_calyx_chromium/+/18323

```
optimization_guide_features.cc
kPageTopicsBatchAnnotations is moved
Do we need to find it and address it elsewhere for this change to be effective?

....

Yeah this got reworked entirely, it's not longer a "feature" we can disable, and it's split across multiple commits.
Not a simple move then so let's let it be.
```

that flag became `kPageVisibilityBatchAnnotations`:

<details><summary>v114</summary>
<p>

![image](https://github.com/uazo/bromite-buildtools/assets/29201891/cbddde70-bd73-42fc-9882-24d4816d5ba8)

</p>
</details> 

<details><summary>v115</summary>
<p>

![image](https://github.com/uazo/bromite-buildtools/assets/29201891/2c6e5873-9b08-4172-a6ba-11f823dfe28e)

</p>
</details> 

please ask, if you like, next time, four eyes are better than two!!!

**Comment by chirayudesai** [comment]
Thank you so much!

I will definitely ask and refer to your patches going forward. Tracking that at https://gitlab.com/CalyxOS/calyxos/-/issues/1771
